app/validators/status_length_validator.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
 class StatusLengthValidator < ActiveModel::Validator
-  MAX_CHARS = 500
+  # Plaza should be inbetween Mastodon and Medium.
+  # Since 1tb is our current maximum storage limit (though it will be expanded at some point) it shouldn't be too much.
+  # Average essay is 500 words. Average Wikipedia essay is 2500 words.
+  # 1000 words is best.
+  MAX_CHARS = 1000
```

Comment:
Plaza should be inbetween Mastodon and Medium.
Since 1tb is our current maximum storage limit (though it will be expanded at some point) it shouldn't be too much.
Average essay is 500 words. My best ever essay is just above 1000 words. Average Wikipedia essay is 2500 words.
Let's do 1000 words.

Reply:
1000 words best.